### DISCUSSION TITLE
My favorite tool of 2020!

### DISCUSSION THREAD

**Discussion by osamaqarem** [tailwindlabs/tailwindcss#3221] [Kind words]
Not only is it a great time saver, but it also taught me a lot about raw CSS. The docs are more than amazing which I must compliment. My projects which use tailwind, are the ones I look forward to working on the most! ♥️ 

Keep rocking 🤘

**Comment by MathGeniusJodie** [comment]
tailwind is pretty life-changing

**Comment by simonswiss** [comment]
Thank you @osamaqarem ❤️

> My projects which use tailwind, are the ones I look forward to working on the most! ♥️

How good is it to go back to a project you haven't touched for 6 months, and be immediately productive without leaving the HTML 🙌🙌
